components/apimgt/org.wso2.carbon.apimgt.rest.api.admin/pom.xml

```diff
@@ -162,6 +162,14 @@
     </dependencies>
     <build>
         <plugins>
+            <!--<plugin>-->
+            <!--<groupId>org.wso2.maven.plugins</groupId>-->
+            <!--<artifactId>swagger2cxf-maven-plugin</artifactId>-->
+            <!--<version>1.0-SNAPSHOT</version>-->
+            <!--<configuration>-->
+            <!--<inputSpec>${project.basedir}/src/main/resources/admin-api.yaml</inputSpec>-->
+            <!--</configuration>-->
+            <!--</plugin>-->
```

Comment:
lets remove the commented out section

Reply:
This would be need when adding a new rest api, we will need to uncomment it and generate the files, Therefore I think it would better to keep it for now

Reply:
+1 for keeping commented.